lib/auth/api.go

```diff
@@ -649,6 +649,9 @@ type ReadWindowsDesktopAccessPoint interface {
 
 	// GetWindowsDesktopService returns a windows desktop host by name.
 	GetWindowsDesktopService(ctx context.Context, name string) (types.WindowsDesktopService, error)
+
+	// CheckOSSDesktopsLimit checks if number of non-AD is in limit for OSS distribution. Returns always true for Enterprise
+	CheckOSSDesktopsLimit(ctx context.Context) (bool, error)
```

Comment:
If there's a way to avoid adding this to the interface and making it an internal implementation detail I think that would be preferred.

Reply:
We have to have consistent implementation on both auth (to manage alerts) and windows desktop service to check things before connection, I think this is the least error-prone to do it. I _think_ this also matches the way suggested by @espadolini previously